pkg/models/errors.go

```diff
@@ -61,4 +61,5 @@ var (
 	ErrPrivateLinkSupportedOnlyForSingleDC      = errors.New("private link is only supported for a single data centre")
 	ErrPrivateLinkSupportedOnlyForAWS           = errors.New("private link is supported only for an AWS cloud provider")
 	ErrImmutableSpec                            = errors.New("resource specification is immutable")
+	ErrOnlySingleConcurrentResizeAvailable      = errors.New("only 1 concurrent resize is allowed")
```

Comment:
Maybe we should change message from "only 1 concurrent resize is allowed" to "only single concurrent resize is allowed"

Reply:
Changed